mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut

```diff
@@ -6,8 +6,9 @@ string highlight   = "\x1b[36m"
 string text        = "\x1b[38:5:152m"
 string adminHeader = "\x1b[94m"
 string ownerHeader = "\x1b[92m"
-
-const FSU_LIST_ROWS = 5
+string error       = "\x1b[38;5;203m"

```

Comment:
Not a big fan of these colours, as I said I'll move this to ConVars and probably make these fit together better.

Reply:
In can make the move to these being ConVars while I'm at it.
I could also set the new colors I added to just the same value as %T, making the colors by default be unchanged?